Gemfile.lock

```diff
@@ -274,7 +274,7 @@ GEM
     rb-fsevent (0.9.5)
     rb-inotify (0.9.5)
       ffi (>= 0.5.0)
-    rdoc (4.2.0)
+    rdoc (4.3.0)
```

Comment:
How come this version got changed?

Reply:
I'm not sure, I didn't change the gem file. Should I change this back to the 4.2.0 version.

Reply:
Yes please change it back. We don't generally update gems unless necessary, in case there's unexpected behavior. The app has so many gems, and those gems have other dependencies, that it would be a nightmare to deal with.